README.md

```diff
@@ -3,8 +3,135 @@
 [![Coverage Status](https://coveralls.io/repos/github/andela/team-odd-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/team-odd-bn-backend?branch=develop)
 [![Maintainability](https://api.codeclimate.com/v1/badges/97cfd183a562e8b5d53f/maintainability)](https://codeclimate.com/github/andela/team-odd-bn-backend/maintainability)
 
-# Barefoot Nomad - Making company travel and accommodation easy and convenient.
+Barefoot Nomad - Making company travel and accomodation easy and convinient.
 =======
 
-## Vision 
+## Vision
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
+
+---
+
+## Documentation
+
+For API documentation, please visit [Barefoot Docs](https://team-odd-bn-backend-staging.herokuapp.com/api-docs/)
+
+## Setting Up The Application
+
+### A. Clone the Application
+
+1. Open your terminal 
+
+2. Run `clone https://github.com/andela/team-odd-bn-backend.git` OR [Download](https://github.com/andela/team-odd-bn-backend/archive/develop.zip) the project
+
+### B. Setting up the environment
+
+1. **Create** a `.env` file to put in all environmental variables
```

Comment:
The project already has `.env.example` so as part of this pull request, you're supposed to rename it to `.env` then all users have to do is add the variables, not create new files.